header/p2p/server.go

```diff
@@ -9,13 +9,21 @@ import (
 	"github.com/libp2p/go-libp2p/core/network"
 	"github.com/libp2p/go-libp2p/core/protocol"
 	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
 
 	"github.com/celestiaorg/go-libp2p-messenger/serde"
 
 	"github.com/celestiaorg/celestia-node/header"
 	p2p_pb "github.com/celestiaorg/celestia-node/header/p2p/pb"
 )
 
+var (
+	serverTracer = otel.Tracer("header/server")
```

Comment:
Just `tracer` is enough since we are in `server.go` and I assume no other tracer exists.